nanshe_workflow/data.py

```diff
@@ -807,23 +807,14 @@ def update(self, dct):
             chunks = tuple(max(c) for c in v.chunks)
             v = v.rechunk(chunks)
 
-            # Salt keys to avoid referencing expired futures
-            # ref: https://github.com/dask/dask/issues/3322
-            v = v.map_blocks(
-                lambda a, u: a,
-                dtype=v.dtype,
-                token="salted",
-                u=uuid.uuid1().bytes
-            )
```

Comment:
Addressed by PR ( https://github.com/dask/dask/pull/3540 ). Hence this workaround can be dropped.